### PR TITLE
Add 2nd outreachy presentation event

### PR DESCRIPTION
**Pull request by mariipia10**

### CODE REVIEW

File: content/events/2022-02-2nd-outreachy-presentation/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: " Outreachy Intern Presentation : Community Onboarding Project"
+date: '2022-03-03'
+days: 1
+tease: ''
+continent: GL
+location: 'Online, Global'
+image: "https://www.outreachy.org/static/css/img/logo-no-background.86e64a85437c.svg"
+location_url: "/events/outreachy-presentation/"
+external_url:
```
Comment (annefou) [inline content/events/2022-02-2nd-outreachy-presentation/index.md:10]: Do you erally need to set location_url? This looks ood to me.

Comment (mariipia10) [reply content/events/2022-02-2nd-outreachy-presentation/index.md:10]: I'm not sure, I've copied the other event template but now revising other templates it isn't used. I'll delete it

**Review by annefou — APPROVED** [review]
It looks good to me now.

**Review by beatrizserrano — APPROVED** [review]
yes, looks good to me too! thanks @mariipia10

**Pull request merged**